datafusion/physical-expr/src/expressions/binary.rs

```diff
@@ -1156,39 +1150,25 @@ impl BinaryExpr {
         array: &dyn Array,
         scalar: &ScalarValue,
     ) -> Result<Option<Result<ArrayRef>>> {
+        let bool_type = &DataType::Boolean;
```

Comment:
changes here were to make the code more readable (so `cargo fmt` put it on the same line)